### PR TITLE
MGDAPI-1632: Update resources for 1M quota

### PR DESCRIPTION
**Pull request by sergioifg94**
# Description

> Link to JIRA: https://issues.redhat.com/browse/MGDAPI-1632

## Verification steps

1. Install RHOAM from this PR
2. Verify that the resources match the new values:
    | Resource | Replicas | Requests CPU | Requests Memory | Limits CPU | Limits Memory |
    | ------------- | ----------- | -------------------- | ------------------------- | --------------- | ------------------- |
    | apicasts | 2 | 0.06 vCPU | 250 MiB | 0.18 vCPU | 300 MiB |
    | listeners | 2 | 0.06 vCPU | 450 MiB | 0.18 vCPU | 500 MiB |
    | workers | 2 | 0.03 vCPU | 60 MiB | 0.09 vCPU | 100 MiB |
    | ratelimit | 2 | 0.02 vCPU | 40 MiB | 0.06 vCPU | 80 MiB |
    | User SSO | 2 | 0.75 vCPU | 1500 MiB | 1.5 vCPU | 1500 MiB |

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: pkg/addon/quota.go [hunk 1]
```diff
@@ -310,79 +310,79 @@ var quotaConfig string = `
...
+				"requests": {
+					"cpu": 0.06,
+					"memory": "250Mi"
+				},
+				"limits": {
+					"cpu": 0.18,
+					"memory": "300Mi"
+				}
+			}
+		},
```
Comment (KevFan) [inline pkg/addon/quota.go:332]:
```suggestion
		"backend_listener": {
			"replicas": 2,
			"resources": {
				"requests": {
					"cpu": 0.06,
					"memory": "450Mi"
				},
				"limits": {
					"cpu": 0.18,
					"memory": "500Mi"
				}
			}
		},
```
Memory values don't seem to match ticket / table in PR for backend listener ? :thinking:

Comment (sergioifg94) [reply pkg/addon/quota.go:332]: I had one job! Good catch, thanks Kevin, I'll fix it now :D

**Comment by KevFan** [comment]
Verified :+1: 

* Quota is 1
image

* Apicast values are as expected in ApiManager CR
image

* Backend Listener values are as expecred in ApiManager CR
image

* Backend Worker values are as expected in ApiManager CR
image

* Rate limit values are as expected in RateLimite Deployment
image

* User SSO values area as expected in Keycload CR
image

/lgtm

**Pull request merged**